### ISSUE TITLE
How does traccar plot on the map?

### ISSUE THREAD

**Issue by younus93** [tananaev/traccar#1466]
Where is the code segment which corresponds to plotting on Openstreetmap?

**Comment by tananaev** [comment]
Map and map controller are responsible for that:

https://github.com/tananaev/traccar/blob/master/web/app/view/Map.js
https://github.com/tananaev/traccar/blob/master/web/app/view/MapController.js

**Comment by younus93** [comment]
Do you use openlayers to build up a map plot? How does it work?

**Comment by tananaev** [comment]
Yes, I use OpenLayers library. There is a lot of documentation about it on the internet.

**Comment by younus93** [comment]
I couldnt understand how you are able to plot the points on the map. Can you please explain the process behind it?

**Comment by tananaev** [comment]
Please ask more specific questions.
